src/concepts/workflows/flow.md

```diff
@@ -0,0 +1,84 @@
+# Workflow Execution and Data Flow
+
+The order of Workflow operation can be split into two concepts:
+
+## Workflow Execution
+
+A Workflow processes Nodes in sequential order:
+
+1. Beginning at the root of the Node tree with either a "Start" or "Intercept" Node.
+2. Every subsequent node in the chain is processed in order. The execution may take the path of another branch based on conditionals.
+3. The Workflow ends either when it reaches an explicit "End" Node or when no more Nodes are available in the chain.
+
+<img alt="Register command SDK." src="/_images/execution_flow.png" center/>
+
+## Workflow Data Flow
+
+Each Node in a Workflow has an input and output data type:
+
+::: info
+The `Choice` and `Code` types are just variations of `String`.
+:::
+
+``` rust
+pub enum NodeDataProp {
```

Comment:
Lets not dump a rust enum here, most of that data is unnecessary and hard to read for users.
This was mostly a reference for you to craft the documentation around it.